addie/utilities/file_handler.py

```diff
@@ -46,3 +46,13 @@ def create_config_parser(self, section_name='Configuration', dictionary=None):
 
         config.write(cfgfile)
         cfgfile.close()
+
+    def check_file_writable(self):
```

Comment:
Change this name to `is_file_writable` or `can_write_file`. I also think it would be clearer to have this as free function in the `file_handler` package rather than as an instance method.